camel/types/enums.py

```diff
@@ -420,11 +420,14 @@ class ModelType(UnifiedModelType, Enum):
     def __str__(self):
         return self.value
 
-    def __new__(cls, value) -> "ModelType":
+    def __repr__(self):
```

Comment:
Lacking this will cause "RecursionError: maximum recursion depth exceeded while calling a Python object" error if we try to print or log this